code/controllers/subsystems/virtual_reality.dm

```diff
@@ -57,14 +57,45 @@
 		ckey = vr_mob.ckey
 		vr_mob.ckey = null
 		vr_mob.old_mob = null
-		vr_mob.languages = list(LANGUAGE_TCB)
+		if(issilicon(vr_mob))
+			var/mob/living/silicon/S = vr_mob
+			S.speech_synthesizer_langs = list(all_languages[LANGUAGE_TCB])
```

Comment:
I suggest making this into a mob proc:
`/mob/proc/vr_exit()`
where you do this stuff